pkg/dashboard/frontend/src/components/logs/FilterSidebar.tsx

```diff
@@ -0,0 +1,185 @@
+import {
+  Sidebar,
+  SidebarContent,
+  SidebarGroup,
+  SidebarGroupAction,
+  SidebarGroupContent,
+  SidebarGroupLabel,
+} from '@/components/ui/sidebar'
+import { Button } from '../ui/button'
+import {
+  Collapsible,
+  CollapsibleContent,
+  CollapsibleTrigger,
+} from '../ui/collapsible'
+import { ChevronRightIcon } from '@heroicons/react/24/outline'
+import { useMemo, type PropsWithChildren } from 'react'
+import { MultiSelect } from '../shared/MultiSelect'
+import { useParams } from '@/hooks/use-params'
+import { useWebSocket } from '@/lib/hooks/use-web-socket'
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '../ui/select'
+
+interface CollapsibleGroupProps extends PropsWithChildren {
+  title: string
+  defaultOpen?: boolean
+}
+
+const CollapsibleGroup = ({
+  title,
+  children,
+  defaultOpen,
+}: CollapsibleGroupProps) => {
+  return (
+    <Collapsible defaultOpen={defaultOpen} className="group/collapsible">
+      <SidebarGroup className="p-0">
+        <SidebarGroupLabel
+          asChild
+          className="flex h-full w-full items-center p-2 text-base font-semibold text-foreground text-gray-600 hover:bg-gray-100 group-data-[state=open]/collapsible:text-foreground"
+        >
+          <CollapsibleTrigger
+            data-testid={`filter-${title.toLowerCase().replace(' ', '-')}-collapsible`}
+          >
+            <ChevronRightIcon className="mr-2 !size-6 transition-transform group-data-[state=open]/collapsible:rotate-90" />
+            <span className="mb-0.5">{title}</span>
+          </CollapsibleTrigger>
+        </SidebarGroupLabel>
+        <CollapsibleContent>
+          <SidebarGroupContent className="p-2">{children}</SidebarGroupContent>
+        </CollapsibleContent>
+      </SidebarGroup>
+    </Collapsible>
+  )
+}
+
+const levelsList = [
+  { value: 'error', label: 'Error' },
+  { value: 'warning', label: 'Warning' },
+  { value: 'info', label: 'Info' },
```

Comment:
Do we want to include 'debug' in this list?

Reply:
We can add these later, to detect these other types from the cmd output we would need to do some investigating